crates/apub/src/objects/post.rs

```diff
@@ -227,7 +228,8 @@ impl ApubObject for ApubPost {
         .community_id(community.id)
         .ap_id(Some(page.id.clone().into()))
         .locked(page.comments_enabled.map(|e| !e))
-        .stickied(page.stickied)
+        .featured_community(page.stickied)
+        .featured_local(Some(false))
```

Comment:
Same here (you can just remove the line).

Reply:
Okay fixed